src/ansys/dpf/post/transient_mechanical_simulation.py

```diff
@@ -0,0 +1,4083 @@
+"""Module containing the ``TransientMechanicalSimulation`` class."""
+from typing import List, Tuple, Union
+import warnings
```

Comment:
Same. I think this file has no changes from the previous implementation?

Reply:
This has been updated.